dht-node/src/index.ts

```diff
@@ -21,9 +21,8 @@ async function main() {
     logger.fatal('Fatal: Database Version incorrect')
     throw new Error('Fatal: Database Version incorrect')
   }
-
-  // eslint-disable-next-line no-console
-  console.log(
+  logger.debug(`dhtseed set by CONFIG.FEDERATION_DHT_SEED=${CONFIG.FEDERATION_DHT_SEED}`)
```

Comment:
```suggestion
  logger.debug('dhtseed set by CONFIG.FEDERATION_DHT_SEED', CONFIG.FEDERATION_DHT_SEED)
```

Reply:
in case of a structured type of the variable, your are right.
but with a simple string or number i preferre the ${} notation by creating the log-msg with the patter: label=value